### PR TITLE
Feature/install material UI

### PR DESCRIPTION
**Pull request by matedyl**

### CODE REVIEW

File: app/app/src/pages/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+                            <Typography variant="body2">Paste a YouTube video link to ASCIIfy it on the fly!</Typography>
+                        </Box>
+                        <Container>
+                            <Typography variant="subtitle1">- or -</Typography>
+                        </Container>
+                        <Box className={classes.introBox}>
+                            <Typography variant="body2">Click to select or drag an image on this box to ASCIIfy
+                                it</Typography>
+                        </Box>
+                        <Container className={classes.donateContainer}>
```
Comment (Esser50K) [inline app/app/src/pages/Layout.tsx:244]: I think it probably looks better if the donation stuff is at the bottom

**Review by Esser50K — COMMENTED** [review]
Thanks for the layouting efforts so far, really starting to get shape!

Some things I couldn't comment on the files themselves because they were pictures.
So the `futureIsHere.gif` file I don't see it being used anywhere, maybe just remove it from the PR, also not sure if all the other pictures are so maybe just double check. Also would prefer if all the thumbnails and all that would not be part of the repo

The ascii background thing I think we can get rid of and just use the background color that YT uses.
To reiterate on that, I don't necessarily want the website to look like it was developed in the 90s but rather have a 90s vibe over it but still be really recognizable. I know this is very subjective so I found these examples which can serve as inspiration [example1](https://www.freepik.com/free-vector/flat-design-90s-nostalgic-birthday-landing-page_19946502.htm) [example2](https://images-na.ssl-images-amazon.com/images/I/61bYvIveXbL.jpg) [example3](https://image.spreadshirtmedia.net/image-server/v1/mp/products/T1459A839MPA4459PT28D192978848FS1458/views/1,width=1200,height=1200,appearanceId=839,backgroundColor=F2F2F2/retro-aesthetic-90s-vibe-vintage-sticker.jpg)

File: app/app/src/App.tsx [hunk 2]
```diff
@@ -1,209 +1,73 @@
...
+            <Route path="/" element={<Layout />}>
+              <Route index element={<Home />} />
+              <Route path="*" element={<NoPage />} />
+            </Route>
+            <Route path="/watch" element={<Layout drawerCollapsed />}>
+              <Route index element={<Watch />} />
+            </Route>
+          </Routes>
+        </BrowserRouter>
+      </ThemeProvider>
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/App.tsx:70]: Yes, `Layout` is the parent with an `Outlet` component inside, which is then replaced with a child component provided to `Layout`

File: app/app/src/App.tsx [hunk 3]
```diff
@@ -1,209 +1,73 @@
...
-        window.addEventListener('click', () => {
-          setShowToast(false);
-        })
-        setShowPlayer(true)
-        setYtUrl("https://www.youtube.com/watch?v=" + v)
-        setShowToast(true)
-      })
+  },
+  typography: {
+    fontFamily: "'Pixelated Times New Roman', 'Times New Roman'",
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/App.tsx:37]: yes, we can also add a generic `serif` type at the end

File: app/app/src/components/Toast.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ interface ToastProps {
 
 function Toast(props: ToastProps) {
     return (
-        <div className='toast-container top-left'>
+        <div className='toast-container bottom-left'>
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/components/Toast.tsx:9]: Yes, it's on the `Watch` page, but I moved to bottom so it doesn't interfere with the new layout

File: app/app/src/components/YoutubeVideoThumbnail.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                height: 0,
+                paddingBottom: "calc(100% / (16/9))",
+            },
+            filter: "blur(1px) saturate(300%)",
+            marginBottom: theme.spacing(1.5),
+        }
+    })
+)
+
+interface YoutubeVideoThumbnailProps {
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/components/YoutubeVideoThumbnail.tsx:29]: ok

File: app/app/src/fonts/fonts.d.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+declare module "*.ttf";
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/fonts/fonts.d.ts:1]: it allows for importing a font as an ESM module

File: app/app/src/pages/Home.tsx [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+    return (
+        <Grid container spacing={3}>
+            {
+                featuredVideos.map(featuredVideo => (
+                    <Grid key={featuredVideo.id} item xs={3}>
+                        <YoutubeVideoThumbnail {...featuredVideo.props} />
+                    </Grid>
+                ))
+            }
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/pages/Home.tsx:144]: ok

File: app/app/src/pages/Home.tsx [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+                thumbnailImage: {
+                    src: '/images/video-thumbnails/8.webp'
+                },
+                avatarImage: {
+                    alt: 'Esser50k',
+                    src: '/images/avatar/Esser50k.jpg'
+                },
+            }
+        }
+    ]
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/pages/Home.tsx:134]: ok. could you branch off this branch so we don't race?

File: app/app/src/pages/Layout.tsx (hunk 1 above)
Comment (matedyl) [reply app/app/src/pages/Layout.tsx:244]: @Esser50K do you mean at the bottom of sidebar or page?

**Comment by matedyl** [comment]
> Thanks for the layouting efforts so far, really starting to get shape!
> 
> Some things I couldn't comment on the files themselves because they were pictures. So the `futureIsHere.gif` file I don't see it being used anywhere, maybe just remove it from the PR, also not sure if all the other pictures are so maybe just double check. Also would prefer if all the thumbnails and all that would not be part of the repo
> 
> The ascii background thing I think we can get rid of and just use the background color that YT uses. To reiterate on that, I don't necessarily want the website to look like it was developed in the 90s but rather have a 90s vibe over it but still be really recognizable. I know this is very subjective so I found these examples which can serve as inspiration [example1](https://www.freepik.com/free-vector/flat-design-90s-nostalgic-birthday-landing-page_19946502.htm) [example2](https://images-na.ssl-images-amazon.com/images/I/61bYvIveXbL.jpg) [example3](https://image.spreadshirtmedia.net/image-server/v1/mp/products/T1459A839MPA4459PT28D192978848FS1458/views/1,width=1200,height=1200,appearanceId=839,backgroundColor=F2F2F2/retro-aesthetic-90s-vibe-vintage-sticker.jpg)

got it! will update the style

futureIsHeregif is used on 404 page - let me know what you think of this one!

**Comment by matedyl** [comment]
@Esser50K the updates are in.

Let's discuss the updates point by point and the remaining color and style updates you have in mind.

Please use the following schedulink for sync call this time: https://cal.com/mateo-product-design

File: app/app/src/pages/Layout.tsx (hunk 1 above)
Comment (matedyl) [reply app/app/src/pages/Layout.tsx]: moved it to the bottom

Comment (Esser50K) [reply app/app/src/pages/Layout.tsx]: bottom of sidebar

**Review by Esser50K — APPROVED** [review]

**Pull request merged**